plugin-manifest.json

```diff
@@ -1,70 +1,70 @@
 {
-"api": {
-"require_startup_execution": false,
-"class_name": "com.applicaster.jwplayerplugin.JWPlayerAdapter"
-},
-"dependency_repository_url": [
-{
-"url": "https://dl.bintray.com/applicaster-ltd/maven_plugins"
-},
-{
-"url": "https://mvn.jwplayer.com/content/repositories/releases/"
-}
-],
-"platform": "android",
-"author_name": "Juan Hernandez",
-"author_email": "j.hernandez@applicaster.com",
-"manifest_version": "1.3.4",
-"name": "JW Player",
-"description": "Android plugin for JWPlayer",
-"type": "player",
-"identifier": "JWPlayer-Plugin",
-"ui_builder_support": true,
-"dependency_name": "com.applicaster:JWPlayerPlugin",
-"dependency_version": "1.3.4",
-"whitelisted_account_ids": [],
-"min_zapp_sdk": "4.3.12-dev",
-"react_native": false,
-"custom_configuration_fields": [
-{
-"type": "text",
-"key": "LICENSE_KEY",
-"tooltip_text": "JW License Key"
-},
-{
-"type": "text",
-"key": "live_ad_url",
-"tooltip_text": "Live Ad Url"
-},
-{
-"type": "text",
-"key": "live_ad_type",
-"tooltip_text": "Live Ad Type"
-},
-{
-"type": "text",
-"key": "live_ad_offset",
-"tooltip_text": "Live Ad Offset"
-},
-{
-"type": "text",
-"key": "vod_midroll_ad_url",
-"tooltip_text": "VOD Midroll URL"
-},
-{
-"type": "text",
-"key": "vod_midroll_offset",
-"tooltip_text": "VOD Midroll Offset"
-},
-{
-"type": "text",
-"key": "vod_preroll_ad_url",
-"tooltip_text": "VOD Preroll URL"
-},
-{
-"type": "text",
-"key": "vod_ad_type",
-"tooltip_text": "VOD Ad Type"
-}
-]
+  "api": {
+    "require_startup_execution": false,
+    "class_name": "com.applicaster.jwplayerplugin.JWPlayerAdapter"
+  },
+  "dependency_repository_url": [
+    {
+      "url": "https://dl.bintray.com/applicaster-ltd/maven_plugins"
+    },
+    {
+      "url": "https://mvn.jwplayer.com/content/repositories/releases/"
+    }
+  ],
+  "platform": "android",
+  "author_name": "Juan Hernandez",
```

Comment:
Add your name instead if you are going to update

Reply:
👍